config/routes.rb

```diff
@@ -6,5 +6,11 @@
   get "up" => "rails/health#show", as: :rails_health_check
 
   # Defines the root path route ("/")
-  # root "posts#index"
+  # Set root to home#index for the base URL
+  root 'home#index'
+
+  # Individual routes for users, their posts, and specific posts
+  get 'users/:user_id', to: 'users#show', as: 'user'
+  get 'users/:user_id/posts', to: 'posts#index', as: 'user_posts'
+  get 'users/:user_id/posts/:id', to: 'posts#show', as: 'user_post'
 end
```

Comment:
- [ ] Ensure you update your routes to show the `users#index` route corresponding to the controller action and the view file as per the requirement

![image](https://github.com/anyars-encarta/encarta-blog-app/assets/81932478/7794510d-f892-43e6-8c2b-6536765a5b55)